pkg/controller/attach_detach.go

```diff
@@ -348,7 +368,7 @@ func (r *ReconcileAttachDetach) deleteFinalizer(azVolumeAttachment *diskv1alpha2
 
 // TODO: reassess if adding additional finalizer to AzVolume CRI is necessary
```

Comment:
Can we remove this commented out code? It's been this way for a while, so it seems like we do not need it.

Reply:
Done 👍